test/parallel/test-http-client-abort-no-agent.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+const net = require('net');
+
+const server = http.createServer(() => {
+  throw new Error('test invalidation');
```

Comment:
@cjihrig are you suggesting to use this?

```js
http.createServer(() => common.fail('whatever'));
```

I thought you wanted to pass it directly to `http.createServer()`

```js
http.createServer(common.fail);
```